subsys/bluetooth/shell/bt.c

```diff
@@ -2891,7 +2891,7 @@ static int cmd_auth_pairing_confirm(const struct shell *shell,
 	return 0;
 }
 
-#if defined(CONFIG_BT_WHITELIST)
+#if defined(CONFIG_BT_FA_LIST)
 static int cmd_wl_add(const struct shell *shell, size_t argc, char *argv[])
```

Comment:
```suggestion
static int cmd_fl_add(const struct shell *shell, size_t argc, char *argv[])
```

Reply:
This PR only changes the public API @LingaoM. We will need further cleanups, but I want to keep those separate.